opentreemap/treemap/views/tree.py

```diff
@@ -118,6 +120,49 @@ def search_tree_benefits(request, instance):
     return context
 
 
+def simplified_tree_benefits(request, instance):
+    unit = 'kg/rok'
+    req = requests.get(settings.ECO_SERVICE_URL)
+    req.raise_for_status()
+    data = req.json()
+
+    descriptions = {
+        'O3': u'Redukcja ozonu (O3)',
+        'NO2': u'Redukcja dwutlenku azotu (NO2)',
+        'SO2': u'Redukcja dwutlenku siarki (SO2)',
+        'PM2.5': u'Redukcja pyłów 2.5 µm (PM2,5)'
+    }
+
+    benefits = {
+        k.replace('.', ''):
+        {
+            'label': descriptions[k],
+            'value': round(v / 1000, 2),
```

Comment:
Why we divide by 1000?

Reply:
The `ecobenefits` returns values in gram units, however grams don't look really meaningful in the frontend. I'll put a comment about this.